src/routes/users/-components/UsersList.tsx

```diff
@@ -3,20 +3,18 @@ import { useSearch, useNavigate } from '@tanstack/react-router';
 import { CodeBlock } from 'ui/codeBlock/CodeBlock';
 import { useQuery } from 'hooks/useQuery/useQuery';
 import { UserSortType } from 'routes/users';
+import { authQueries } from 'api/actions/auth/auth.queries';
 
 export const UsersList = () => {
   const { sort, page } = useSearch({ from: '/users/' });
   const navigate = useNavigate();
 
-  const { data: usersResponse, isFetched: areUsersFetched } = useQuery(
-    'getUsersList',
-    { page: page.toString() || undefined },
-    {
-      select: (data) => {
-        return { ...data, users: data.users.sort((a, b) => (sort === 'desc' ? +b.id - +a.id : +a.id - +b.id)) };
-      },
+  const { data: usersResponse, isFetched: areUsersFetched } = useQuery({
+    ...authQueries.list({ page: page.toString() }),
+    select: (data) => {
+      return { ...data, users: data.users.sort((a, b) => (sort === 'desc' ? +b.id - +a.id : +a.id - +b.id)) };
```

Comment:
I'm always surprised when I see this + - + : + - + :D looks a bit like https://en.wikipedia.org/wiki/Brainfuck

Reply:
I think it's worth looking into whether we need the ability to sort here at all and if so, is sorting by id the way to go.

Reply:
We might want to change / delete that in the future :D